srcpkgs/azote/template

```diff
@@ -1,26 +1,27 @@
 # Template file for 'azote'
 pkgname=azote
-version=1.7.10
-revision=2
+version=1.8.1
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3>=3.4 python3-setuptools python3-gobject python3-Pillow gtk+3 feh python3-send2trash xrandr grim slurp ImageMagick python3-yaml"
+depends="python3-setuptools python3-gobject python3-Pillow gtk+3"
 short_desc="Wallpaper & color manager for Sway, i3 and other WMs"
 maintainer="Piotr Miller <nwg.piotr@gmail.com>"
 license="BSD-3-Clause, GPL-3.0-or-later"
 homepage="https://github.com/nwg-piotr/azote"
 distfiles="${homepage}/archive/v${version}.tar.gz"
-checksum=e9ecd2104689c05fb2a37bb160543d80e9fd245c5274ab16fc021f400d738d35
+checksum=d8b2f5f4e351e72d9c58f9d5fc9c9e2526087e57e5553f6defbd163c78727905
 
 post_install() {
 	vmkdir usr/bin
 	vmkdir usr/share/${pkgname}
 	vmkdir usr/share/applications
 
 	vcopy "dist/${pkgname}.svg" usr/share/${pkgname}
-	vcopy "dist/${pkgname}-arch" usr/share/${pkgname}
+	vcopy "dist/${pkgname}" usr/share/${pkgname}
```

Comment:
There's no value in copying this script to `usr/share` and then symlinking to it from `usr/bin`, just `vcopy` right into `usr/bin` and drop the symlink in line 26.

Reply:
Done